test/unit/requestData.test.js

```diff
@@ -0,0 +1,54 @@
+import { pagination } from '../../src/models/requestData'
+import { describe, it, expect } from 'vitest'
+
+// // Tech Debt: we should write some tests around the requestData.js file
```

Comment:
Worth capturing a Trello card for this tech debt and including the link in the comments